health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FacilityPopup.js

```
@@ -0,0 +1,41 @@
+  const config = facilityMappingConfig();
+
+  return (
+    <>
+      <Card type="secondary" />
+      <PopUp
+        onClose={onClose}
+        heading={`${t(`MICROPLAN_ASSIGNMENT_FACILITY`)} ${details?.additionalDetails?.name}`}
+        children={[
+          <InboxSearchComposer
+            configs={config} // dummy config
+          ></InboxSearchComposer>,
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)